### PR TITLE
Chore/ground nav promos width

### PR DESCRIPTION
**Pull request by spaceninja**
## Overview

While testing the updated Ground Nav in WordPress, I noticed that if the promo content was too short, and there was only one promo showing, it should shrink and look odd. This PR remedies that by applying a min-width to the promo container.

## Screenshots

Screenshot 2023-09-07 at 11 34 07 AM

## Testing

Review the "One Feature" story on the Ground Nav component on the preview environment. It should look appropriate even though it now has short content.

### CODE REVIEW

File: src/utilities/size/demo/block-size.twig [hunk 1]
```diff
@@ -4,7 +4,7 @@
       full (in a 150px tall container)
     </div>
   </div>
-  <div style="height:150px;">
+  <div style="height:150px;" class="t-alternate">
```
Comment (spaceninja) [inline src/utilities/size/demo/block-size.twig:7]: Unrelated change that didn't get included in the last PR that makes the block size utilities demo easier to understand.

**Review by Paul-Hebert — DISMISSED** [review]
LGTM!
